docs-new/docs/graphql/core/migrations/advanced/auto-apply-migrations.mdx

```diff
@@ -1,17 +1,18 @@
-<div class="meta"
-description="Auto-apply migrations and metadata when the server starts"
-keywords="hasura, docs, auto-apply, migration, metadata, server">
-
-</div>
+---
+sidebar_label: Auto-apply migrations/metadata
```

Comment:
Minor mismatch in sidebar_label

`Auto-apply migrations/metadata when the server starts`